### PR TITLE
[src, scripts] fix typos in the comment.

### LINKED ISSUE #3262: kaldi-stream.h not found
See the comment below
https://github.com/kaldi-asr/kaldi/blob/e3a9844aa35e2ad5437bb7e5cbfe8326fd28e635/src/util/kaldi-table.h#L81-L83

But there is no file called `kaldi-stream.h`

### LINKED ISSUE #3264: Error in the comment for wxfilename and rxfilename
See the comment below
https://github.com/kaldi-asr/kaldi/blob/0cc941f7e5fd519a9ffba8c85adafee6d7db0b83/src/util/kaldi-io.h#L55-L72

For the wxfilename
> (3) A pipe: e.g.  "gunzip -c /tmp/abc.gz |"

which is actually an **rxfilename**.

For the rxfilename
> (3) A pipe: e.g. "| gzip -c > /tmp/abc.gz"

which is a **wxfilename**.

### LINKED ISSUE #3265: Error in the comment for WspecifierOptions
See the comment below
https://github.com/kaldi-asr/kaldi/blob/0cc941f7e5fd519a9ffba8c85adafee6d7db0b83/src/util/kaldi-table.h#L69-L70

But the default is actually **not flush**

https://github.com/kaldi-asr/kaldi/blob/0cc941f7e5fd519a9ffba8c85adafee6d7db0b83/src/util/kaldi-table.h#L117

### LINKED ISSUE #3444: Inconsistent naming convention in simple-decoder.cc
## Problem

See the code
https://github.com/kaldi-asr/kaldi/blob/893181fdfdbd6ed3818c789eaa79fc4f238ac276/src/decoder/simple-decoder.cc#L217

The variable `queue_` is a local variable but ending with an underscore `_` .

## Expected

Change it to
```
 std::vector<StateId> queue; 
```

### PR DESCRIPTION
**Pull request by csukuangfj**
This pullrequest

- fix #3262
- fix #3264
- fix #3265
- fix #3444

### CODE REVIEW

**Comment by danpovey** [comment]
I'll leave this PR open until you accumulate a few more of these, as you seem to have been finding a lot of these types of issues lately.

**Comment by naxingyu** [comment]
There are some cases where the name of the binary/script shown in the usage is incorrect.

File: egs/wsj/s5/steps/compute_cmvn_stats.sh [hunk 1]
```diff
@@ -81,7 +81,7 @@ required="$data/feats.scp $data/spk2utt"
 
 for f in $required; do
   if [ ! -f $f ]; then
-    echo "make_cmvn.sh: no such file $f"
+    echo "compute_cmvn_stats.sh: no such file $f"
```
Comment (kkm000) [inline egs/wsj/s5/steps/compute_cmvn_stats.sh]: I think it should better be just `$0`.

Comment (csukuangfj) [reply egs/wsj/s5/steps/compute_cmvn_stats.sh]: @kkm000 

Done.

**Comment by csukuangfj** [comment]
@danpovey 
I've collected many changes in a single commit and please have a review of them.

Thanks

**Comment by danpovey** [comment]
Thanks, please also include this
```
a09:src: git diff doc
diff --git a/src/doc/tree_externals.dox b/src/doc/tree_externals.dox
index 01f8dc433..cc753ef39 100644
--- a/src/doc/tree_externals.dox
+++ b/src/doc/tree_externals.dox
@@ -344,7 +344,7 @@ has value 0 (i.e. the leftmost HMM-state).  Assuming the answer is "yes", the ne
 is "SE 2 [ 220 221 222 223 ]", which is asking whether the phone to the right is one of various
 forms of the phone "M" (a rather unintuitive question to ask, since we're
 in the leftmost HMM-state); if yes, we ask "SE 0 [ 104 105 106 107... 286 287 ]" which is
-a question about the phone to the right; if yes, then the pdf-id is 5 ("CE 5") and if
+a question about the phone to the left; if yes, then the pdf-id is 5 ("CE 5") and if
 no, 696 ("CE 696").
 \verbatim
 s3# copy-tree --binary=false exp/tri1/tree - 2>/dev/null | head -100

```

I won't merge this right away-- keep looking for typos, if you ahve time, and I'll merge them in one go, in a few days.

File: src/feat/feature-common.h [hunk 2]
```diff
@@ -89,7 +89,7 @@ class ExampleFeatureComputer {
      @param [out] feature  Pointer to a vector of size this->Dim(), to which
          the computed feature will be written.
   */
-  void Compute(BaseFloat signal_log_energy,
+  void Compute(BaseFloat signal_raw_log_energy,
```
[the remark this answers is not in the document]
Comment (csukuangfj) [reply src/feat/feature-common.h]: @kkm000 
There are two reasons:

1. the comment uses it.
https://github.com/kaldi-asr/kaldi/blob/893181fdfdbd6ed3818c789eaa79fc4f238ac276/src/feat/feature-common.h#L75

2. There is a function with the same name:
https://github.com/kaldi-asr/kaldi/blob/893181fdfdbd6ed3818c789eaa79fc4f238ac276/src/feat/feature-common.h#L61

File: src/matrix/kaldi-vector.h [hunk 3]
```diff
@@ -355,7 +355,7 @@ class VectorBase {
   friend class CuVector<Real>;
  protected:
   /// Destructor;  does not deallocate memory, this is handled by child classes.
-  /// This destructor is protected so this object so this object can only be
+  /// This destructor is protected object so this object can only be
```
[the remark this answers is not in the document]
Comment (csukuangfj) [reply src/matrix/kaldi-vector.h]: @kkm000 
done.

**Comment by csukuangfj** [comment]
@danpovey 

Done!

File: src/doc/tree_externals.dox [hunk 4]
```diff
@@ -341,7 +341,7 @@ the first "real" question in the tree.  We can see from the SE questions above i
 applies when the central phone takes values 5 through 19, which are
 various versions of the phone AA; and question is asking whether the pdf-class (key -1)
 has value 0 (i.e. the leftmost HMM-state).  Assuming the answer is "yes", the next question
-is "SE 2 [ 220 221 222 223 ]", which is asking whether the phone to the right is one of various
+is "SE 2 [ 220 221 222 223 ]", which is asking whether the phone to the left is one of various
```
Comment (danpovey) [inline src/doc/tree_externals.dox]: In this case, "right" is actually correct.  (2 is the right-most position in the 3-phone window).

Comment (csukuangfj) [reply src/doc/tree_externals.dox]: @danpovey 

I just applied the patch you posted above. I will revert it back tomorrow.

Comment (danpovey) [reply src/doc/tree_externals.dox]: I think line 347 is the one which should have been changed.

Comment (csukuangfj) [reply src/doc/tree_externals.dox]: @danpovey 

yes, you're correct!

Fixed.

**Pull request merged**